File: framec_tests/csharp/FrameLang/FrameEvent.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

#nullable disable
namespace FrameLang
{
    public class FrameEvent
    {
    public string _message { get; set; }
    public Dictionary<string, object> _parameters { get; set; }
    public object _return { get; set; }

    public FrameEvent(string message, Dictionary<string, object> parameters)
    {
    
        _message = message;
        _parameters = parameters;
        
    }
    public FrameEvent(){}
    }
}

File: framec_tests_old/java/FrameLang/FrameEvent.java
package framec_tests.java.FrameLang;

import java.util.*;

public class FrameEvent {
    public FrameEvent(String message, Map<String, Object> parameters) {
        this._message = message;
        this._parameters = parameters;
        this._return = null;
        return;
    }

    public FrameEvent() {
    }

    public String _message;
    public Map<String, Object> _parameters;
    public Object _return;


}
